### PR TITLE
Fix check for style translucency

### LINKED ISSUE #11300: Cesium3DTileStyle - alpha value not respected for loaded tiles
**CesiumJS versions**
Issue present in CesiumJS version 1.105 & 1.105.1. (No issues in cesiumjs version 1.92) Only tested in versions: 1.105, 1.105.1, 1.92

**Expected behaviour:**

When changing the style of a Cesium3DTileset using either of the following:

```javascript
    tileset.style = new Cesium.Cesium3DTileStyle({
      color: 'rgba(255,255,255,0.5)'
    });
```
```javascript
    tileset.style = new Cesium.Cesium3DTileStyle({
      color: 'color("white", 0.5)'
    });
```

I expect that all existing / newly loaded tiles to respect the assigned alpha value. Regardless of if the style has been added before or after the initial render of a Cesium3DTileset.

**Actual behaviour**
When changing the alpha value of a Cesium3DTileset, existing tiles use the old alpha value. Newly loaded tiles use the new alpha value.

The alpha value is respected if the style has been added prior to the model being added to the scene. If the alpha value is changed after this, the issue occurs:

Tile_Alpha_Issue

Sandcastle example:

By showing the tiles content volume, we can see that older tiles aren't affected by the alpha value change.

[Sandcastle](https://sandcastle.cesium.com/#c=lVNNb9swDP0rRLCDDQRK99FiyJJga7dDgQE7JOguPkSRGUeYLAYSnSAt/N8nSw7ibN1hFxvk4+Mj+WxF1jMcNB7RwRwsHuEBvW5q8RRzWTFSMX4gy1JbdMVoDC+FBWB0LmSmZ8IqxWLrqP5JzpR9IssL2+afCltYgwysDXrkS+yk9Xvp0HIYYCuNx4ixO/U6iRBAeZSaz3Lp9f7rKsFR9pHsFx+CxzL7cPPx7q5TBRg0ESVummq5o2O3T5C8p8aW2lZPZJoagwa7BiMr3UR4hRbF3ulasz6gF7Iss75bPix8JqpXdA0NlT2fDF6feLjCsoOzM60FJVntIAsnJJenQ6jgFRkUhqps/a0DwJDspj+LTOHNS2S0665TC90hC7uUtlTSc+CG6VdUVQbvG2aywd5lOO3q4kFnbzRhDJnaofqFZQ7zRW/FlVc9HEfW20F5rP3PzXtOt6UhN4W1qzYye3d7O4bL40bc5utU2KZTtYBh1n8pBm9xGz7aNGMbv8PReDSL+OKs+FnXe3IMjTOZEBPGem8ko59smrAPC+V9EgOYTYbUWakPoMv5K/8IKCO9D8i2MWapn7EYLWaTUP8XtbfwxwGdkaeubPd28T0lhRCzSQhfZzKR2Uj3R+ff)

Browser:
Tested on Chrome (Version 113.0.5672.127) & Edge (Version 113.0.1774.50)

Operating System:
Windows 10 Enterprise - version 21H2

### PR DESCRIPTION
**Pull request by j9liu**
Fixes #11300.

This PR corrects the check in `Model.applyColorAndShow`. The specs didn't catch it because there wasn't a unit test for models without feature tables, so one such unit test was added.

[Local sandcastle for testing.](http://localhost:8080/Apps/Sandcastle/index.html#c=lVNNb9swDP0rRLCDDQRK99FiyJJga7dDgQE7JOguPkSRGUeYLAYSnSAt/N8nSw7ibN1hFxvk4+Mj+WxF1jMcNB7RwRwsHuEBvW5q8RRzWTFSMX4gy1JbdMVoDC+FBWB0LmSmZ8IqxWLrqP5JzpR9IssL2+afCltYgwysDXrkS+yk9Xvp0HIYYCuNx4ixO/U6iRBAeZSaz3Lp9f7rKsFR9pHsFx+CxzL7cPPx7q5TBRg0ESVummq5o2O3T5C8p8aW2lZPZJoagwa7BiMr3UR4hRbF3ulasz6gF7Iss75bPix8JqpXdA0NlT2fDF6feLjCsoOzM60FJVntIAsnJJenQ6jgFRkUhqps/a0DwJDspj+LTOHNS2S0665TC90hC7uUtlTSc+CG6VdUVQbvG2aywd5lOO3q4kFnbzRhDJnaofqFZQ7zRW/FlVc9HEfW20F5rP3PzXtOt6UhN4W1qzYye3d7O4bL40bc5utU2KZTtYBh1n8pBm9xGz7aNGMbv8PReDSL+OKs+FnXe3IMjTOZEBPGem8ko59smrAPC+V9EgOYTYbUWakPoMv5K/8IKCO9D8i2MWapn7EYLWaTUP8XtbfwxwGdkaeubPd28T0lhRCzSQhfZzKR2Uj3R+ff)

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @j9liu!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/fix-style-translucency-check/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

File: CHANGES.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 ##### Fixes :wrench:
 
 - Fixed the error report "rectanglePromise.then is not a function" that occurred when using `viewer.flyTo` to navigate to an ImageryLayer. [#11392](https://github.com/CesiumGS/cesium/pull/11392)
+- Fixed a bug where `Model` would not respond to different alpha values in a `Cesium3DTileStyle`. [#11399](https://github.com/CesiumGS/cesium/pull/11399)
```
Comment (ggetz) [inline CHANGES.md:10]: We've started differentiating changes between `@cesium/widgets` and `@cesium/engine`. Could you please merge in main and nest this under  the `@cesium/engine` header?

**Review by ggetz — COMMENTED** [review]
Awesome, thanks for the fix and the additional unit test @j9liu!

File: CHANGES.md (hunk 1 above)
Comment (j9liu) [reply CHANGES.md]: Oops! Forgot about that, thanks

**Comment by j9liu** [comment]
@ggetz just pushed a commit to fix the changelog!

**Comment by ggetz** [comment]
Thank you again @j9liu!

**Pull request merged**